native/src/components/DatePicker.tsx

```diff
@@ -64,9 +64,18 @@ export type DatePickerProps = {
   error?: string
   modalOpen: boolean
   setModalOpen: (open: boolean) => void
+  placeholderDate?: DateTime
 }
 
-const DatePicker = ({ title, date, setDate, error, modalOpen, setModalOpen }: DatePickerProps): ReactElement => {
+const DatePicker = ({
+  title,
+  date,
+  setDate,
+  error,
+  modalOpen,
+  setModalOpen,
+  placeholderDate,
```

Comment:
🙃  ~~Using a default value here also allows you to get rid of the very unspecific `dateToUse` naming.~~
```suggestion
  placeholderDate = DateTime.now(),
```

Edit: Applying the suggestion above makes this unnecessary.